app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,14 @@
+class MoviesController < ApplicationController
+  def show
+    @movie = Movie.find(params[:id])
+    @actors = @movie.order_age
+  end
+
+  def new
+
+  end
+
+  def create
```

Comment:
I can see this is where you left off - good progress so far through US 3! I encourage you to come back and try to finish this out when you have time, just for practice. 

If/when you do, I would suggest moving this `create` method to a different controller. Mostly because, what are we _creating_? If we're creating a new `Movie` instance, then sure - but the user story is asking us to add an actor to a movie; in other words we're creating an `ActorMovie` instance. Since that's the model we'd create, we should probably do that in a controller that would control those instances - something called `actor_movies_controller`.